contrib/packaging/rpm/ansible-kubespray.spec

```diff
@@ -0,0 +1,60 @@
+%global srcname ansible_kubespray
+
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+Name:           ansible-kubespray
```

Comment:
it would be nice to keep the ansible prefix, as a generic package associated info.